### PR TITLE
change accountType to androidAccountType and convert to enum class in dart

### PR DESCRIPTION
**Pull request by lidongze91**
Minor Polish for android account type.

### CODE REVIEW

File: lib/contacts_service.dart [hunk 1]
```diff
@@ -222,13 +223,25 @@ class Contact {
...
+  AndroidAccountType accountTypeFromString(String androidAccountType) {
+    if (androidAccountType == null) return null;
+    if (androidAccountType.contains("com.google")) {
+      return AndroidAccountType.google;
+    } else if (androidAccountType.contains("com.whatsapp")) {
+      return AndroidAccountType.whatsapp;
+    } else if (androidAccountType.contains("com.facebook")) {
+      return AndroidAccountType.facebook;
+    }
+    return AndroidAccountType.other;
```
Comment (mehmetf) [inline lib/contacts_service.dart]: Make a comment that there are many other account types on Android (such as Samsung etc) and we do not currently support them.

File: lib/contacts_service.dart [hunk 2]
```diff
@@ -222,13 +223,25 @@ class Contact {
...
       this.middleName,
       this.prefix,
       this.suffix,
       this.birthday,
     ].where((s) => s != null));
   }
+
+  AndroidAccountType accountTypeFromString(String androidAccountType) {
+    if (androidAccountType == null) return null;
+    if (androidAccountType.contains("com.google")) {
```
Comment (mehmetf) [inline lib/contacts_service.dart]: .startsWith() might be more accurate?

File: lib/contacts_service.dart (hunk 1 above)
Comment (lidongze91) [reply lib/contacts_service.dart]: Done.

File: lib/contacts_service.dart (hunk 2 above)
Comment (lidongze91) [reply lib/contacts_service.dart]: Done.

**Comment by mehmetf** [comment]
@lukasgit For a second pass. 

This is a breaking change and thus should be documented in the change log accordingly. Please increment the version as well.

**Pull request merged**